components/content/src/page.rs

```diff
@@ -329,6 +338,7 @@ Hello world"#;
             &config,
             InsertAnchor::None,
             &HashMap::new(),
+            Arc::new(Caches::default()),
```

Comment:
applies elsewhere but [`Arc` implements `Default`](https://doc.rust-lang.org/stable/std/sync/struct.Arc.html#impl-Default-for-Arc%3CT%3E) for this case:

```suggestion
            Arc::default(),
```